aerolyzer/image_restriction_functions.py

```diff
@@ -0,0 +1,237 @@
+'''
+Aerolyzer
+March 2, 2017
+Image Restriction Function File
+Description: This file contains all functions for the verifying image restrictions.
+'''
+
+'''
+import libraries
+'''
+import re
+import exifread
+
+class imageRestrictionFunctions(object):
+    'Class containing all image restriction functions'
+
+    def __init__(self):
+        pass
```

Comment:
Consider when initializing this class, to load the data from the config file (as data within in that file is a required input for most of these functions). In that manner, you can reduce the parameter lists on the functions and the dependency on the user to load the config file (better software design).